docs/content/overview/cdc-pipeline.md

```diff
@@ -97,5 +97,5 @@ For example, we can use this yaml file to define a pipeline:
 ```yaml
 pipeline:
   name: mysql-to-kafka-pipeline
-  pipeline.global.parallelism: 1
+  global.parallelism: 1
```

Comment:
Just 'parallelism', not 'global.parallelism'

Reply:
@joyCurry30 ,  Thanks, I will remove it from `cdc-pipeline.md` and mysql's `pipeline-test.yaml`